doc/user/content/sql/join.md

```diff
@@ -29,13 +29,20 @@ expect.
 
 {{< diagram "join-type.svg" >}}
 
+### table_ref
+
+{{< diagram "table-ref.svg" >}}
+
 Field | Use
 ------|-----
 _select&lowbar;pred_ | The predicating [`SELECT`](../select) clauses you want to use, e.g. `SELECT col_ref FROM table_ref...`. The _table&lowbar;ref_ from the _select&lowbar;pred_ is the left-hand table.
 **NATURAL** | Join table expressions on all columns with the same names in both tables. This is similar to the `USING` clause naming all identically named columns in both tables.
-_join&lowbar;type_ | The type of `JOIN` you want to use _(`INNER` is implied default)_.
-_table&lowbar;ref_ | The table expression you want to join, i.e. the right-hand table.
-**USING (** _col&lowbar;ref..._ **)** | If the join condition does not require table-level qualification (i.e. joining tables on columns with the same name), the columns to join the tables on. For example, `USING (customer_id)`.
+**LATERAL** | Indicates that the following subquery or table function call can refer to columns from the left-hand side of the join. See [`LATERAL` subqueries](#lateral-subqueries) below.
+_join\_type_ | The type of `JOIN` you want to use _(`INNER` is implied default)_.
+_select_stmt_ | A [`SELECT` statement](/sql/select).
```

Comment:
Weird--you know why this doesn't need to escape the underscore between `select` and `stmt`? It renders correctly so I'm stumped

Reply:
Gee, I guess Markdown parsers have gotten pretty smart. I added the escape because it was weirding me out.